docs/topics/rules-important-links.adoc

```diff
@@ -5,7 +5,7 @@
 
 * {ProductShortName} Javadoc: http://windup.github.io/windup/docs/latest/javadoc
 * {ProductShortName} forums: https://developer.jboss.org/en/windup
```

Comment:
Not urgent but I would check this forum link with Phil or Jonathan. It looks like it has not been used since 2019